src/main.js

```diff
@@ -1,12 +1,21 @@
-import cli from 'cli';
+import { getConfig } from 'cli';
 import Linter from 'linter';
 import log from 'logger';
 
 import 'babel-polyfill';
 
-export function createInstance({config=cli.argv, runAsBinary=false} = {}) {
+
+export function isRunFromCLI(_module=module) {
+  return require.main === _module;
+}
+
+export function createInstance({
+  config=getConfig(isRunFromCLI()).argv, runAsBinary=false,
```

Comment:
It's for testing largely; coverage reporters will complain if we don't check the logic we use to check that we are running as the top-level module.